src/Sylius/Bundle/CoreBundle/Tests/Responses/Expected/error/not_found.json

```diff
@@ -0,0 +1,4 @@
+{
+    "code": 404,
+    "message": @string@
```

Comment:
you can actually enclose these placeholders in `"`. It still works.
This has the added benefit of being valid json, so can be interpreted/formatted by an IDE.